dockerfiles/platform-build-base/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:18.04 as build
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
I think I read somewhere it wasn't recommended to set this globally, rather than on a particular command, since any install in the container would also have it set.  That is only likely to be developers, so may not be a big issue.

Reply:
It's only an issue if you expect derived containers to be used to launch interactive shells. Which we don't